packages/config/src/layer2s/orderly.ts

```diff
@@ -0,0 +1,78 @@
+import { EthereumAddress, UnixTime } from '@l2beat/shared-pure'
+
+import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
+import { opStack } from './templates/opStack'
+import { Layer2 } from './types'
+
+const discovery = new ProjectDiscovery('orderly')
+
+const upgradeability = {
+  upgradableBy: ['ProxyAdmin'],
+  upgradeDelay: 'No delay',
+}
+
+export const orderly: Layer2 = opStack({
+  daProvider: 'Celestia',
+  discovery,
+  display: {
+    name: 'Orderly Network',
+    slug: 'orderly',
+    warning:
+      'Fraud proof system is currently under development. Users need to trust the block proposer to submit correct L1 state roots.',
+    description:
+      'Orderly is an Optimium based on the OP Stack and using Celestia DA. It serves as a ledger for all transaction data, user data balances, and trading data retrieved from the Orderbook and order-related services.',
+    purposes: ['DeFi'],
+    links: {
+      websites: ['https://orderly.network/'],
+      apps: [],
+      documentation: [
+        'https://orderly.network/docs/build-on-evm/building-on-evm',
+      ],
+      explorers: ['https://explorer.orderly.network/'],
+      repositories: [],
+      socialMedia: [
+        'https://twitter.com/OrderlyNetwork',
+        'https://discord.gg/orderlynetwork',
+        'https://medium.com/@orderlynetwork',
+        'https://t.me/OrderlyNetworkAnnouncements',
+      ],
+    },
+  },
+  upgradeability,
+  l1StandardBridgeEscrow: EthereumAddress(
+    '0xe07eA0436100918F157DF35D01dCE5c11b16D1F1',
+  ),
+  rpcUrl: 'https://rpc.orderly.network',
+  sequencerAddress: EthereumAddress(
+    discovery.getContractValue('SystemConfig', 'batcherHash'),
+  ),
+  inboxAddress: EthereumAddress('0x08aa34cc843ceebcc88a627f18430294aa9780be'),
+  genesisTimestamp: new UnixTime(1696566432),
+
+  l2OutputOracle: discovery.getContract('L2OutputOracle'),
+  portal: discovery.getContract('OptimismPortal'),
+  isNodeAvailable: false,
```

Comment:
why false? at most it should be under review. btw: https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/mainnet/orderly.yaml, https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/extra/genesis/mainnet/orderly.json.gz